Jetcaster/app/src/main/java/com/example/jetcaster/ui/player/PlayerScreen.kt

```diff
@@ -230,6 +249,94 @@ private fun PlayerContentTableTop(
     }
 }
 
+@Composable
+private fun PlayerContentBook(
+    uiState: PlayerUiState,
+    bookPosture: DevicePosture.BookPosture,
+    onBackPress: () -> Unit
+) {
+    val hingePosition = with(LocalDensity.current) { bookPosture.hingePosition.left.toDp() }
+    val hingeWidth = with(LocalDensity.current) { bookPosture.hingePosition.width().toDp() }
+
+    Column(
+        modifier = Modifier
+            .fillMaxSize()
+            .verticalGradientScrim(
+                color = MaterialTheme.colors.primary.copy(alpha = 0.50f),
+                startYPercentage = 1f,
+                endYPercentage = 0f
+            )
+            .systemBarsPadding()
+            .padding(horizontal = 8.dp)
+    ) {
+        TopAppBar(onBackPress = onBackPress)
+
+        Row(modifier = Modifier.fillMaxSize()) {
+            // Content for the left part of the screen
+            PlayerContentBookLeft(hingePosition, uiState)
```

Comment:
Nit: This shouldn't be `Left` and `Right`, this should be `Start` and `End`.

Reply:
Good point, I wanted to do another PR to reuse the book layout on larger screen.
I can put it there.